spec/routes/db_spec.rb

```diff
@@ -0,0 +1,598 @@
+
+    it 'must return status 200 and include records data' do
+      visit "/db/phone_numbers/#{phone_number.id}"
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_xpath(header_xpath)
+    end
+  end
+
+  context 'GET /db/street_addresses' do
+    let!(:street_address) { Ronin::DB::StreetAddress.find_or_create_by(address: 'Address', city: 'City', state: 'State', zipcode: '00-000', country: 'Country') }
+    let(:xpath)           { "//a[text()='#{street_address}'][@href='/db/street_addresses/#{street_address.id}']" }
+
+    it 'must return status 200 and include link to a street address in the body' do
+      visit '/db/street_addresses'
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_xpath(xpath)
+    end
+  end
+
+  context 'GET /db/street_addresses/:id' do
+    let!(:street_address) { Ronin::DB::StreetAddress.find_or_create_by(address: 'Address', city: 'City', state: 'State', zipcode: '00-000', country: 'Country') }
+    let(:header_xpath)    { "//h1[text()='Street Address: #{street_address}']" }
+
+    it 'must return status 200 and include records data' do
+      visit "/db/street_addresses/#{street_address.id}"
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_xpath(header_xpath)
+    end
+  end
+
+  context 'GET /db/organizations' do
+    let!(:organization) { Ronin::DB::Organization.find_or_import('org_name') }
+    let(:xpath)         { "//a[text()='#{organization}'][@href='/db/organizations/#{organization.id}']" }
+
+    it 'must return status 200 and include link to a organization in the body' do
+      visit '/db/organizations'
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_xpath(xpath)
+    end
+  end
+
+  context 'GET /db/organizations/:id' do
+    let!(:organization) { Ronin::DB::Organization.find_or_import('org_name') }
+    let(:header_xpath)  { "//h1[text()='Organization: #{organization}']" }
+
+    it 'must return status 200 and include records data' do
+      visit "/db/organizations/#{organization.id}"
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_xpath(header_xpath)
+    end
+  end
+
+  context 'GET /db/people' do
+    let!(:person) { Ronin::DB::Person.find_or_import('full name') }
+    let(:xpath)   { "//a[text()='#{person}'][@href='/db/people/#{person.id}']" }
+
+    it 'must return status 200 and include link to a person in the body' do
+      visit '/db/people'
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_xpath(xpath)
+    end
+  end
+
+  context 'GET /db/people/:id' do
+    let!(:person)      { Ronin::DB::Person.find_or_import('full name') }
+    let(:header_xpath) { "//h1[text()='Person: #{person}']" }
+
+    it 'must return status 200 and include records data' do
+      visit "/db/people/#{person.id}"
+
+      expect(page.status_code).to eq(200)
+      expect(page).to have_xpath(header_xpath)
+    end
+  end
+end
```

Comment:
Add a `after` block to destroy any of the models created for the specs.

```ruby
after do
  Ronin::DB::Person.destroy_all
  Ronin::DB::StreetAddress.destroy_all
end
```

You could add `after` blocks to each `context`, but that might be repetitious, so maybe better to just add one single `after` block.